build/LegacyRuntimes.props

```diff
@@ -0,0 +1,57 @@
+<Project>
+  <PropertyGroup Condition=" '$(IncludeLegacySharedFramework)' == 'true' ">
+    <Legacy110CoreSetupChannel>release/1.1.0</Legacy110CoreSetupChannel>
```

Comment:
The additional one ships in the CLI itself when it is set.

The legacy one is used for CLI testing purposes only.